lib/manageiq/api/common/users.rb

```diff
@@ -0,0 +1,70 @@
+module ManageIQ
+  module API
+    module Common
+      module Users
```

Comment:
@syncrou 
Should this just be a class called Users and we return an instance of the user after we have parsed the headers.
The user instance would store the decoded hash and allow the user to access the key value pairs

Reply:
Ahh yes, we were discussing an `OpenStruct` class to return the keys as attributes.